README.md

```diff
@@ -28,15 +30,15 @@ registers_client = RegistersClient::RegisterClientManager.new({
 })
 ```
 
-The `RegisterClientManager` maintains individual instances of `RegisterClient` for each register you access. 
+The `RegisterClientManager` maintains an invidual instance of [`RegisterClient`](#registerclient) for each time you access a register by calling [`get_register`](#getregister).
```

Comment:
Perhaps something like:

_The `RegisterClientManager` maintains individual instances of `RegisterClient` for each register you access via the `get_register` method_

I propose this because the instances are cached, so you won't necessarily get a new instance back each time you call `get_register`, which is how I read the proposed update.

Reply:
That's fine, I'll change to your suggestion unless @arnau has any queries.

Reply:
I'm fine with @karlbaker02 proposal.

Reply:
Great, changed!